### PR TITLE
Disable Vanilla Chunk GC

### PR DESCRIPTION
**Pull request by uRyanxD**

### CODE REVIEW

**Comment by uRyanxD** [comment]
Rebased

**Comment by uRyanxD** [comment]
Rebased

**Comment by uRyanxD** [comment]
Rebased

**Comment by uRyanxD** [comment]
Rebased

**Comment by uRyanxD** [comment]
Rebased

**Comment by uRyanxD** [comment]
Rebased

**Review by mechoriet — APPROVED** [review]

File: patches/server/0091-Disable-Vanilla-Chunk-GC.patch [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+diff --git a/src/main/java/net/minecraft/server/WorldServer.java b/src/main/java/net/minecraft/server/WorldServer.java
+index 0700b5aec26a6a4ba4db4d3c66b8b15c295c5a03..ab107574c8027debefb437f1605789b1d48d5593 100644
+--- a/src/main/java/net/minecraft/server/WorldServer.java
++++ b/src/main/java/net/minecraft/server/WorldServer.java
+@@ -964,6 +964,7 @@ public class WorldServer extends World implements IAsyncTaskHandler {
+             this.chunkProvider.saveChunks(flag, iprogressupdate);
+             } // PandaSpigot
+             // CraftBukkit - ArrayList -> Collection
++            /* // PandaSpigot start - Disable Vanilla Chunk GC
```
Comment (mechoriet) [inline patches/server/0091-Disable-Vanilla-Chunk-GC.patch:16]: maybe keep comment and comment block seperate lines

like:

```java
// PandaSpigot start - Disable Vanilla Chunk GC
/*
```

```java
*/
// PandaSpigot end
```

makes the diff a bit larger but also perhaps more readable if there are more changes around it in future

File: patches/server/0091-Disable-Vanilla-Chunk-GC.patch [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+             Collection arraylist = this.chunkProviderServer.a();
+             Iterator iterator = arraylist.iterator();
+ 
+@@ -974,6 +975,7 @@ public class WorldServer extends World implements IAsyncTaskHandler {
+                     this.chunkProviderServer.queueUnload(chunk.locX, chunk.locZ);
+                 }
+             }
++            */ // PandaSpigot end
```
Comment (mechoriet) [inline patches/server/0091-Disable-Vanilla-Chunk-GC.patch:24]: maybe keep comment and comment block seperate lines

**Comment by uRyanxD** [comment]
applied suggestions

**Review by mechoriet — APPROVED** [review]

**Pull request merged**